src/app/AppErrorMessage.vue

```diff
@@ -7,8 +7,8 @@
       <KIcon
         class="mb-3"
         icon="warning"
-        color="var(--black-500)"
-        secondary-color="var(--yellow-300)"
+        color="var(--tmp-black-500)"
+        secondary-color="var(--tmp-yellow-300)"
```

Comment:
Whenever using custom properties, I'd highly recommend providing a fallback value. 

To be honest, I'd recommend switching this implementation to utilizing HEX values or shared JS variables instead of CSS custom properties altogether; users of the app can't customize the value so there's really no point. 

Comments apply to the whole app/PR

Reply:
I’ve avoided this problem by introducing a dedicated WarningIcon component and hard-coding its colors in https://github.com/kumahq/kuma-gui/pull/1314/commits/696c3661e92813b3eaa65a344f363cc56879f9ee.